swesmith/profiles/javascript.py

```diff
@@ -219,7 +219,7 @@ def log_parser(self, log: str) -> dict[str, str]:
 class Axiosef36347f(JavaScriptProfile):
     owner: str = "axios"
     repo: str = "axios"
-    commit: str = "ef36347fb559383b04c755b07f1a8d11897fab7f"  # Replace with a specific commit hash
+    commit: str = "ef36347fb559383b04c755b07f1a8d11897fab7f"
```

Comment:
I've made a separate PR (#134) to remove `axios/axios` because it's in SWE-bench_Multilingual.

Reply:
Added `eval_sets` to indicate this!